domains/org.eclipse.lyo.tools.domainmodels/oslcDomainSpecifications.xml

```diff
@@ -82,18 +88,21 @@
     <resourceProperties id="_posD4ZUdEeq-KoPaR9_Cfg" name="label" description="May be used to provide a human-readable version of a resource's name." occurs="zeroOrOne" valueType="XMLLiteral">
       <propertyDefinition href="vocabulary.xml#//@vocabularies[label='RDFS']/@properties[name='label']"/>
     </resourceProperties>
+    <resourceProperties id="_uuAX4JUdEeq-KoPaR9_Cfg" name="label" description="May be used to provide a human-readable version of a resource's name." occurs="zeroOrOne" valueType="String">

```

Comment:
Note that this property was not added, It was a modification of what we have on line 94 in old version. We include a description of the property. We also moved the definiition to be next to the "label" that is of type XmlLiteral.